plugins/splunk-on-call/src/components/Escalation/EscalationPolicy.tsx

```diff
@@ -24,12 +30,28 @@ import { useApi, Progress } from '@backstage/core';
 import { Alert } from '@material-ui/lab';
 import { User } from '../types';
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      maxHeight: '400px',
+      overflow: 'auto',
+    },
+    subheader: {
+      backgroundColor: 'white',
```

Comment:
Best use a color from the theme palette here, or dark mode is gonna look strange